atom/browser/api/atom_api_window.cc

```diff
@@ -338,16 +338,30 @@ bool Window::IsFullscreen() {
   return window_->IsFullscreen();
 }
 
-void Window::SetBounds(const gfx::Rect& bounds) {
-  window_->SetBounds(bounds);
+void Window::SetBounds(mate::Arguments* args) {
```

Comment:
The `args->Length()` is in fact the length of all parameters, so it should be `args->Length() == 2` in this case. Our C++ wrapping is not very perfect and checking optional parameter is a bit error-prone, so we usually just ignore the error.